docker-compose.yml

```diff
@@ -55,5 +55,20 @@ services:
       replicas: ${WORKER_COUNT:-2}
     restart: unless-stopped
 
+  web:
+    build: .
+    env_file:
+      - .env       # loads OPENWEATHER_API_KEY (and any other vars)
+    volumes:
+      - ./uploads:/app/uploads   # so your app.py / weather_service can write/read
+      - .:/app                   # if you want code auto-reloads in dev
+    command: flask run --host=0.0.0.0
+    ports:
+      - "5000:5000"
+    depends_on:
+      - redis
+      - api
+      - worker
+
```

Comment:
What is this service for? Seems that it duplicates existing functions, no?

Reply:
> What is this service for? Seems that it duplicates existing functions, no?

It is for debugging only and can be deleted